viacoin-blockchain-ubuntu/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dnsLabelPrefix": {
+      "value": "viacoinnode"
+    },
+    "adminUsername": {
+      "value": "azureuser"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "installMethod": {
+      "value": "From_Source"
+    },    
+    "vmSize": {
+      "value": "Standard_D1"
+    }
```

Comment:
I have edited the azuredeploy.json and azuredeploy.parameter.json to your suggestions..
However it doesn't pass the Tavis-ci check.

my bet is that it goes wrong because of the root and sasToken.
The guide is very confusing.

Travis gives back this error. Could you please look at the template again and show me what I did wrong and type out what I should correct. Just how you typed out the refference endpoint as you did here https://github.com/Azure/azure-quickstart-templates/pull/2214#discussion_r70146668
I would appreciate that.

This is the error Travis gives me back. 
"error": "InvalidRequestContent : The request content was invalid and could not be deserialized: 'Could not find member 'parameters' on object of type 'TemplateResource'. Path 'properties.template.resources[5].parameters', line 1, position 4761.'.\n"